### DISCUSSION TITLE
F10C2654-5AA1-4563-8826-95915DB0B685

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#21] [General]
# F10C2654-5AA1-4563-8826-95915DB0B685

K2-138 is a K-type main sequence star about 664.55 light-years away in the constellation Aquarius. It is quite a large and compact system, where all of its six planets (3 superterrans, 3 neptunians) orbit within 0.23 AU from their star, in other words, all of them are closer to their star than Mercury is to the Sun!

https://celestia.mobi/addon?item=F10C2654-5AA1-4563-8826-95915DB0B685

**Comment by exocritic5457** [comment]
I'm sorry, but these textures look incredibly rushed

**Reply by exocritic5457** [reply]
Thankfully this aged like milk a few hours ago.

**Reply by TheLostProbe** [reply]
Karistus has been revamping a lot of lower-quality add-ons recently and it's great
